zulipterminal/ui.py

```diff
@@ -259,6 +259,10 @@ def keypress(self, size: Tuple[int, int], key: str) -> Optional[str]:
             # Show help menu
             self.controller.show_help()
             return key
+        elif is_command_key('MARKDOWN_HELP', key):
+            # Show markdown help menu
+            self.controller.show_markdown_help(self.model.server_url)
```

Comment:
We can get `server_url` in a similar way in `show_markdown_help`. In that case, do we need to pass it as an argument here?